.github/workflows/release-updates.yml

```diff
@@ -0,0 +1,25 @@
+# Automatically updates "vX" branches based on GitHub releases. To cut a new
+# release, use the GitHub UI where you enter a tag name and release name of
+# "vX.Y.Z". See https://github.com/jupyterhub/action-k8s-namespace-report/releases.
+---
+name: Release updates
+
+on:
+  release:
+    types: [published, edited]
+
+jobs:
+  actions-tagger:
+    runs-on: windows-latest
```

Comment:
Do you know if this also works on ubuntu? In general I'd expect Linux systems to start-up quicker than windows.

Reply:
I think it doesn't, I recall investigating this before because i was uneasy of a windows based action ^^

Reply:
That's a shame....maybe we can rewrite it ourselves in future :smile:

Reply:
Haha yes, i considered it! With this github action we are half the way :)